blosc/frame.c

```diff
@@ -1025,7 +1025,7 @@ int64_t frame_from_schunk(blosc2_schunk *schunk, blosc2_frame_s *frame) {
 uint8_t* get_coffsets(blosc2_frame_s *frame, int32_t header_len, int64_t cbytes, int32_t *off_cbytes) {
   if (frame->coffsets != NULL) {
     if (off_cbytes != NULL) {
-      *off_cbytes = *(int32_t *) (frame->coffsets + BLOSC2_CHUNK_CBYTES);
+      *off_cbytes = sw32_(frame->coffsets + BLOSC2_CHUNK_CBYTES);
```

Comment:
Also prefer to re-read the entire blosc header here too.

Reply:
+1